src/inference/include/openvino/runtime/intel_npu/properties.hpp

```diff
@@ -53,5 +53,13 @@ static constexpr ov::Property<uint64_t, ov::PropertyMutability::RO> device_total
  */
 static constexpr ov::Property<uint32_t, ov::PropertyMutability::RO> driver_version{"NPU_DRIVER_VERSION"};
 
+/**
+ * @brief [Only for NPU compiler]
+ * Type: std::string
+ * Set various parameters supported by the NPU compiler.
+ * @ingroup ov_runtime_npu_prop_cpp_api
+ */
+static constexpr ov::Property<std::string> compilation_mode_params{"NPU_COMPILATION_MODE_PARAMS"};
```

Comment:
There is also an method `isPublic`, it is set to false for now:

https://github.com/openvinotoolkit/openvino/blob/84530e7f33ddc66ebbc9912319f47502b268a852/src/plugins/intel_npu/src/al/include/intel_npu/al/config/compiler.hpp#L159

However, I am not sure if the method is used in the current implementation

Reply:
Changed to TRUE now, Thanks!